src/etc/sysctl.d/30-vyos-router.conf

```diff
@@ -83,6 +83,16 @@ net.ipv4.conf.default.ignore_routes_with_linkdown=1
 net.ipv6.conf.all.ignore_routes_with_linkdown=1
 net.ipv6.conf.default.ignore_routes_with_linkdown=1
 
+# Disable IPv6 interface autoconfigurationnable packet forwarding for IPv6
+net.ipv6.conf.all.autoconf=0
+net.ipv6.conf.default.autoconf=0
+net.ipv6.conf.*.autoconf=0
```

Comment:
Does it really work?
```
vyos@r14:~$ sudo sysctl -w net.ipv6.conf.*.autoconf=0
sysctl: cannot stat /proc/sys/net/ipv6/conf/*/autoconf: No such file or directory
vyos@r14:~$ 

```

Reply:
It's valid syntax according to the manual page https://man7.org/linux/man-pages/man5/sysctl.d.5.html

Reply:
Thanks!